src/app/lib/analyticsUtils/frontpage.js

```diff
@@ -0,0 +1,36 @@
+import deepGet from '../utilities/deepGet';
+
+export const getPageIdentifier = frontpageData =>
+  deepGet(['metadata', 'analyticsLabels', 'counterName'], frontpageData) ||
+  'unknown.page';
+
+const guidRegex =
+  '([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})';
```

Comment:
Are we sure of this regex format, or does it just match our 3 cases?

Reply:
I have looked though all the services and confirmed that they all match this regex.